docs/content/downloads.md

```diff
@@ -10,6 +10,17 @@ Rclone is single executable (`rclone`, or `rclone.exe` on Windows) that you can
 simply download as a zip archive and extract into a location of your choosing.
 See the [install](https://rclone.org/install/) documentation for more details.
 
+## Release {{% version %}} {#release} OS requirements
```

Comment:
The `{#release}` here is an anchor for linking to so `https://url/#anchor`  - it belongs at the end and it needs to be different, so either remove it or put it at the end and call it `{#osrequirements}` say then we'll be able to link to it on the forum.